test/integration/cli/test/index.test.js

```diff
@@ -579,6 +579,31 @@ describe('CLI Usage', () => {
       expect(stripAnsi(stdout).trim()).toBeFalsy()
     })
 
+    test('Allow retry if port 3000 is already in use', async () => {
+      const port = 3000
+      let output = ''
+
+      const appOne = await runNextCommandDev(
+        [dirBasic, '-p', port],
+        undefined,
+        {}
+      )
+      const appTwo = await runNextCommandDev(
+        [dirBasic, '-p', port],
+        undefined,
+        {
+          onStderr(msg) {
+            output += stripAnsi(msg)
+          },
+        }
+      )
+
+      await killApp(appOne)
+      await killApp(appTwo)
```

Comment:
```suggestion
      try {
      const appOne = await runNextCommandDev(
        [dirBasic, '-p', port],
        undefined,
        {}
      )
      const appTwo = await runNextCommandDev(
        [dirBasic, '-p', port],
        undefined,
        {
          onStderr(msg) {
            output += stripAnsi(msg)
          },
        }
      )
      } finally {
      await killApp(appOne).catch(console.error)
      await killApp(appTwo).catch(console.error)
      }
```
We should wrap these in a try/finally to cleanup properly if one crashes or similar.

Reply:
<picture data-single-emoji=":done:" title=":done:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/done/556da3823bdfab37.png" alt=":done:" align="absmiddle"></picture>